README.md

```diff
@@ -243,7 +243,7 @@ dependencies:
 
 ### Versioning
 
-The regression test notebooks follow semantic versioning:
+The regression test notebooks try to follow semantic versioning:
```

Comment:
https://giphy.com/gifs/starwars-star-wars-yoda-26FmQ6EOvLxp6cWyY

(I get it, though, it's not completely straightforward to apply semantic versioning to these notebooks)